### PR TITLE
fix : code editor's text size should change at least one each time when you try to change it using shortcuts

### PR DESCRIPTION
**Pull request by Raphael2048**
Fix:code editor text size should change at least one each time, don't… mind how you setting your display scale 

_Bugsquad edit: Closes #18575_

### CODE REVIEW

File: editor/code_editor.cpp [hunk 1]
```diff
@@ -650,12 +650,12 @@ void CodeTextEditor::_text_editor_gui_input(const Ref<InputEvent> &p_event) {
 }
 
 void CodeTextEditor::_zoom_in() {
-	font_resize_val += EDSCALE;
+	font_resize_val += EDSCALE < 1.0f ? 1 : EDSCALE;
```
Comment (groud) [inline editor/code_editor.cpp]: You change that by `MAX(EDSCALE, 1.0f)` (and also down there) ? 
That would make the code easier to read.

**Comment by groud** [comment]
Sorry to bother again but could you squash the two commit together ?
See the [PR workflow](http://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase). :)

**Comment by raphael10241024** [comment]
sorry about that, first time to do this

**Comment by groud** [comment]
> sorry about that, first time to do this

No problem, everybody goes through those first steps ^^

**Comment by hpvb** [comment]
Cherry picked into 3.0.3

**Pull request merged**